scripts/treasury/treasury.py

```diff
@@ -0,0 +1,37 @@
+from brownie import (
+    TreasuryAction,
+    Router
+)
+from brownie import accounts, interface
+
+EnvironmentConfig = {
+    "cDAI": '0x5d3a536e4d6dbd6114cc1ead35777bab948e3643',
+    "WETH": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+}
+
+def main():
```

Comment:
Can you make a fork environment for mainnet testing in general?

Reply:
done